pages/shapes.tsx

```diff
@@ -2,8 +2,38 @@ import { useState } from 'react';
 import styled from 'styled-components';
 
 import Image from '../public/colouring-images/Shapes.svg';
+import { ImageHeader } from '../components/ImageHeader';
 import { PaletteControl } from '../components/PaletteControl';
 import { Colour } from '../types';
+import { SaveModal } from '../components/SaveModal';
+import { VisuallyHidden } from '../components/VisuallyHidden';
+
```

Comment:
It feels a bit annoying that all the code has to be repeated in each image. I can't remember why it was hard to have the images as svgs that got propped into a shared component?

This design choice also makes it difficult for darts to add their own new colouring pics.

Reply:
It is annoying. It was difficult to dynamically import an image as a React component, so I went with duplicating last time and didn't change it in this PR. I'll make an issue so it gets done another time.